roles/slurm-stats/tasks/main.yml

```diff
@@ -0,0 +1,36 @@
+---
```

Comment:
How come just this bit of slurm-stats.yml is broken out as a role? Should this be in the new slurm-tools repo? Or just inlined back into slurm-stats.yml given this is "just" a demo?

Reply:
Moved into collection.